arangod/Replication2/ReplicatedLog/LogLeader.cpp

```diff
@@ -252,14 +252,14 @@ void replicated_log::LogLeader::executeAppendEntriesRequests(
         // Capture a weak pointer `parentLog` that will be locked
         // when the request returns. If the locking is successful
         // we are still in the same term.
+        auto currentCommitIndex = request.leaderCommit;
+        auto lowestIndexToKeep = request.leaderCommit;
```

Comment:
I think there's a c&p error here:
```suggestion
        auto lowestIndexToKeep = request.lowestIndexToKeep;
```